main.js

```diff
@@ -0,0 +1,25 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
+
+// const hamburger = document.querySelector(".hamburger");
+// const navMenu  = document.querySelector(".nav-menu");
+
+// hamburger.addEventListener("click", () => {
+//   hamburger.classList.toggle("active");
+//   navMenu.classList.toggle("active");
+// })
+
+//  document.querySelectorAll(".nav-link").forEach((n) => n.addEventListener("click", () => {
+//    hamburger.classList.remove("active");
+//    navMenu.classList.remove("active");
+//  }))
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly remove this commented code block, they make your code base not clean and professional.